src/transformers/configuration_transfo_xl.py

```diff
@@ -80,7 +81,7 @@ class TransfoXLConfig(PretrainedConfig):
             adaptive (:obj:`boolean`, optional, defaults to :obj:`True`):
                 use adaptive softmax
             tie_weight (:obj:`boolean`, optional, defaults to :obj:`True`):
-                tie the word embedding and softmax weights
+                tie the word embedding and softmax weights. THIS PARAMETER IS DEPRECATED.
```

Comment:
@sgugger - not sure if this is a good way of saying that a parameter is deprecated

Reply:
It should be hidden in the kwargs if deprecated, ideally.

Reply:
True - I delete it